src/mobile/src/ui/components/MultiTextInput.js

```diff
@@ -9,6 +9,8 @@ import { width, height } from 'libs/dimensions';
 import { Styling } from 'ui/theme/general';
 import { getThemeFromState } from 'shared-modules/selectors/global';
 import AmountTextInput from './AmountTextInput';
+import { /*MAX_TAG_LENGTH,*/ MAX_MESSAGE_LENGTH } from 'shared-modules/libs/iota/utils';
```

Comment:
ESLint is showing an error here:

```

/workdir/src/mobile/src/ui/components/MultiTextInput.js
--
  | 12:56  error  Absolute imports should come before relative imports  import/first

```
This can be solved by moving this line above line 11